### PR TITLE
hls-parser add apple low latency hls types

### PR DESCRIPTION
**Pull request by joeflateau**
hls-parser supports apple's low latency hls

### CODE REVIEW

File: types/hls-parser/index.d.ts [hunk 1]
```diff
@@ -94,6 +109,10 @@ export namespace types {
                 segments?: readonly Segment[];
                 prefetchSegments?: readonly PrefetchSegment[];
                 source?: string;
+                lowLatencyCompatibility?: LowLatencyCompatibility;
+                partTargetDuration?: number;
+                renditionReports?: RenditionReport[];
```
Comment (screendriver) [inline types/hls-parser/index.d.ts:114]: Should we declare all arrays as `readonly` like all other arrays here?

Comment (joeflateau) [reply types/hls-parser/index.d.ts:114]: can't hurt for consistency I guess, I don't believe these are actually read-only though

Comment (screendriver) [reply types/hls-parser/index.d.ts]: Usually they are not. But for consistency and prevent (accidental) mutation `readonly` would be nice 🙂

**Review by screendriver — APPROVED** [review]

**Comment by joeflateau** [comment]
Ready to merge

**Pull request merged**